Fix: removed 'submit-row' from detach_alias.html

albanbochsler opened the pull request:
## Description

The css class `submit-row` causes a weird behavior while extending the `delete_confirmation.html` template – resulting in 404 every time a Alias is detached in the structure view. 

***

* [ ] I have added or modified the tests when changing logic
* [x] I have followed [the conventional commits guidelines](https://www.conventionalcommits.org/) to add meaningful information into the changelog
* [ ] I have read the [contribution guidelines ](https://github.com/django-cms/django-cms/blob/develop/CONTRIBUTING.rst)

fsbraun commented:
@albanbochsler I need to update the test dependency. Do not worry about failed tests for now.

Adding the `submit-row` class obviously is a great idea. I am not sure I understand the issue with 404, though.

albanbochsler commented:
> @albanbochsler I need to update the test dependency. Do not worry about failed tests for now.
> 
> Adding the `submit-row` class obviously is a great idea. I am not sure I understand the issue with 404, though.

Hi @fsbraun, it seems that the submit input needs a additional css class in order to be rendered correctly in the modal footer. I've reverted my changes and added the class `deletelink` to the submit. What I've meant before is that without the proper indication the submit link will force another dialog rather than closing the modal. As far as I understand the class `submit-row` is a js selector that sets the buttons in the modal footer and interprets the link classes.
